### DISCUSSION TITLE
Virtualized Select 虚拟化选择器 fit-input-width

### DISCUSSION THREAD

**Discussion by accardo** [element-plus/element-plus#18069] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select-v2`

## Description

el-select组件有 这个属性fit-input-width 可以控制 下拉框的宽度是否与输入框相同， 虚拟列表组件没有，希望加一下

### Additional comments

_(empty)_

**Comment by kooriookami** [comment]
虚拟组件的下拉宽度无法精确计算
